setup.bat

```diff
@@ -1,5 +1,7 @@
 cd rg_speech_to_text
-python3.8 -m venv venv
+python -m venv venv
```

Comment:
this does not work on some machines as in case a different python version is the default, it will result in an error e.g. python 3.9 or 2.7 , .... we have to be explicit
usually all python versions are also mapped to a numbered version with the default install, otherwise the user that does not have that would edit the file and takes the responsibility of the user python version

Reply:
In case someone has only Python 3.8 installed, there exists only python and no python3.8. 

I don't know then what is the solution which works in the case when we don't have old python, but python 3.8.
Do we have to make a discovery code for all the python names which exists, then intelligently select the correct version and rename the python.exe or make link with python3.8 ?